examples/features/metadata_interceptor/client/main.go

```diff
@@ -29,18 +29,35 @@ import (
 
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/metadata"
 
 	pb "google.golang.org/grpc/examples/features/proto/echo"
 )
 
 var addr = flag.String("addr", "localhost:50051", "the address to connect to")
 
 func callUnaryEcho(ctx context.Context, client pb.EchoClient) {
-	resp, err := client.UnaryEcho(ctx, &pb.EchoRequest{Message: "hello world"})
+	var header, trailer metadata.MD
+	resp, err := client.UnaryEcho(
+		ctx,
+		&pb.EchoRequest{Message: "hello world"},
+		grpc.Header(&header),
+		grpc.Trailer(&trailer))
+
```

Comment:
Go style does not impose line length restrictions on code lines, and it is recommended to not break-up code lines unless it significantly improves readability. See: https://google.github.io/styleguide/go/guide#line-length

You could simply write this as:
```golang
	resp, err := client.UnaryEcho(ctx, &pb.EchoRequest{Message: "hello world"}, grpc.Header(&header), grpc.Trailer(&trailer))
	if err != nil {
		log.Fatalf("UnaryEcho: %v", err)
	}
```

Or if you don't like such long lines, you could do:
```
	opts := []grpc.CallOption{grpc.Header(&header), grpc.Trailer(&trailer))}
	resp, err := client.UnaryEcho(ctx, &pb.EchoRequest{Message: "hello world"}, opts...)
	if err != nil {
		log.Fatalf("UnaryEcho: %v", err)
	}

```

Reply:
Done